render_configs.sh

```diff
@@ -2,7 +2,7 @@ ENV=${1:-dev}
 VAULT_TOKEN=${2:-$(cat "$HOME"/.vault-token)}
 
 VAULT_ADDR="https://clotho.broadinstitute.org:8200"
-COMMON_VAULT_PATH="secret/dsde/terra/kernel/$ENV/common"
+COMMON_VAULT_PATH="secret/dsde/terra/kernel/perf/common"
```

Comment:
Why this change? I would expect us to still want render_configs to be generic.

Reply:
As I understand it, the only testrunner account that exists is the perf one. Before we were only using the perf account, but the names were written as if it was a generic SA that existed in all environments. This change makes the code more explicit that only `perf` is supported.

I'll update the environment variable name to also be perf specific.